### PR TITLE
bump instance size for cloud demux of NovaSeq-size runs

### PR DESCRIPTION
**Pull request by tomkinsc**
DNAnexus: mem1_ssd2_x36 -> mem2_ssd1_v2_x48

### CODE REVIEW

**Comment by tomkinsc** [comment]
We set the applet [in ansible](https://github.com/broadinstitute/sabetilab-remote-config/blob/master/settings_manager.yml.template#L34) for the uploader laptops (in a crontab job, really), or in my shell script on the Broad cluster.

Good point about the redundancy; I'll remove the threads and ram params from the launder in favor of the WDL. Unfortunately we'll still need to set instance size here since the WDL task doesn't know which size instance it should run on beyond its singular default value.

**Comment by dpark01** [comment]
Agreed. But can you also increase the default task shape in the wdl runtime (even though it shouldn’t matter for auto launched jobs) and maybe nudge up the ram per tile and records in ram in the nova case (though I’m not totally sure by how much, a safe guess might be by 4x each since we’re bumping ram by 4x) in the wdl?

**Comment by dpark01** [comment]
three more requested changes:
1. can we change the illumina_demux task's optional input param `minimumQuality` [from 10 to unbound](https://github.com/broadinstitute/viral-pipelines/blob/master/pipes/WDL/tasks/tasks_demux.wdl#L54) (ie, not specified)? I think we've been using this totally wrong
2. can we change the default `minimumBaseQuality` [from 25 to 10](https://github.com/broadinstitute/viral-pipelines/blob/master/pipes/WDL/tasks/tasks_demux.wdl#L49) (this is the min Q for the index cycles
3. we need to increase the dnanexus [default timeout policy](https://github.com/broadinstitute/viral-pipelines/blob/master/pipes/dnax/dx-extras.json#L6) from 9h to ... 18 maybe?

**Comment by dpark01** [comment]
hm. well I learned that 1.8TB disk isn't enough for a 700GB tarball. and that more threads is helping the speed. let's:
- increase the dx_instance_type to `mem3_ssd2_v2_x32`
- increase the task disk size to 2625 (LOCAL disks have to be multiples of 375 on GCE)
- increase the task cpu/ram to 32 cpus and 200GB RAM (corresponds to an `n1-highmem-32` on GCE)
- maxReadsInRamPerTile=1200000 (this is the Picard default)
- maxRecordsInRam=3000000 (this is 6x higher than Picard default)
- let novaseq thread count be 32

this is all assuming that this [empirically goes well this morning](https://platform.dnanexus.com/projects/Bq29k680jy1JF3gvkk3Gjf11/monitor/analysis/Fp65kb00jy18v0gB1P94Xk7q).

File: pipes/dnax/dx-launcher/demux_launcher.yml [hunk 1]
```diff
@@ -177,7 +164,7 @@ runSpec:
         fi
         for i in $(seq "$lane_count"); do
           folder2=$(printf "%s/%s/reads/L%d" "$folder" "$run_id" $i)
-          runcmd="dx run $demux_workflow_id -i stage-0.flowcell_tgz=$run_tarball -i illumina_demux.lane=$i -i illumina_demux.maxReadsInRamPerTile=$max_reads_in_ram_per_tile -i illumina_demux.maxRecordsInRam=$max_records_in_ram -i illumina_demux.minimumBaseQuality=$min_base_quality -i illumina_demux.threads=$demux_threads $sequencing_center_input --folder $folder2 --instance-type illumina_demux=$demux_instance_type --name demux:$run_id:L$i -y --brief"
+          runcmd="dx run $demux_workflow_id -i stage-0.flowcell_tgz=$run_tarball -i illumina_demux.lane=$i --folder $folder2 --instance-type illumina_demux=$demux_instance_type --name demux:$run_id:L$i -y --brief"
```
Comment (dpark01) [inline pipes/dnax/dx-launcher/demux_launcher.yml:167]: probably want to keep the `$sequencing_center_input` param in there

Comment (tomkinsc) [reply pipes/dnax/dx-launcher/demux_launcher.yml]: Yes; mea culpa

**Pull request merged**